document-picture-in-picture/main.js

```diff
@@ -0,0 +1,90 @@
+
+const videoPlayer = document.getElementById("player");
+const playerContainer = document.getElementById("container");
+let togglePipButton;
+let pipActive = false;
+let pipWindow;
```

Comment:
I think you can also remove this one and simply have `const pipWindow` in your functions.

Reply:
Again, much better! Fixed in next commit.